Makefile

```
@@ -66,3 +66,7 @@ docker-build-startos:
 	cd docker && \
 	docker compose build mostro-startos
 
+mutation-test:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)